### DISCUSSION TITLE
CSS style templates

### DISCUSSION THREAD

**Discussion by DeeliN221** [marp-team/marp#296] [Q&A] [answered]
Marp has a nice but not fascinating default theme, user can define their own style template, but it's rather tricky for CSS newbies. 

Why not list out some commonly used themes that can reproduce the layout of PowerPoint? 
Ohmyzsh has a similar need and they offer a built-in theme library.

**Comment by yhatt** [comment, marked as answer]
Marp team only has provided a simple theme `default`, and some of theme examples as built-in themes. It does not mean that all of users want a bloated Marp module that has a huge collection of layouts, so we leave making and curating more useful themes to our community.

You could find out themes from [GitHub's `marp-themes` topic](https://github.com/topics/marp-themes), [`Show and tell` discussion threads](https://github.com/orgs/marp-team/discussions/categories/show-and-tell), and so on.

You've described about oh-my-zsh, and it is also a project from community. As same as that, we hope someone [make a repository just like awesome-marp as community-driven project](https://github.com/sindresorhus/awesome/blob/main/awesome.md).

Related: #285

**Reply by DeeliN221** [reply]
Thank you for your instructions. 
I stared some Marp theme repos, and is going to do some modifications based on them. 
Though I do really need someone to implement the PowerPoint slide layout because I am not familiar with CSS and HTML stuff.
